ui/components/multichain/pages/send/__snapshots__/send.test.js.snap

```diff
@@ -172,33 +172,43 @@ exports[`SendPage render and initialization should render correctly even when a
             class="mm-box multichain-send-page__recipient"
           >
             <div
-              class="mm-box tabs"
+              class="transform-gpu"
             >
-              <ul
-                class="mm-box tabs__list mm-box--display-flex mm-box--gap-0 mm-box--justify-content-flex-start mm-box--background-color-background-default"
+              <div
+                class="flex flex-row gap-0 justify-start bg-default tabs__list top-0 z-[2]"
+                role="tablist"
```

Comment:
At the moment we don’t have the polymorphic prop in the new box. I’ll add it, and we can update this HTML afterward. In the meantime, I’ve added ARIA labels to ensure accessibility.